Source/ACE.Server/Command/Handlers/DeveloperContentCommands.cs

```diff
@@ -2418,6 +2417,79 @@ public static void ExportSQLLandblock(Session session, string param)
             CommandHandlerHelper.WriteOutputInfo(session, $"Exported {sql_folder}{sql_filename}");
         }
 
+        public static void ExportDiscordLandblock(Session session, string param)
+        {
+            var variationId = session.Player.Location.Variation;
+            ushort landblockId = 0;
+
+            if (!string.IsNullOrEmpty(param) && !param.ToLower().Contains("landblock"))
+            {
+                if (!ushort.TryParse(Regex.Match(param, @"[0-9A-F]{4}", RegexOptions.IgnoreCase).Value, NumberStyles.HexNumber, CultureInfo.InvariantCulture, out landblockId))
+                {
+                    CommandHandlerHelper.WriteOutputInfo(session, $"{param} not a valid landblock");
+                    return;
+                }
+            }
+            else
+            {
+                if (!ushort.TryParse(Regex.Match(session.Player.CurrentLandblock.Id.ToString(), @"[0-9A-F]{4}", RegexOptions.IgnoreCase).Value, NumberStyles.HexNumber, CultureInfo.InvariantCulture, out landblockId))
+                {
+                    CommandHandlerHelper.WriteOutputInfo(session, $"{session.Player.CurrentLandblock.Id.ToString()} not a valid landblock");
+                    return;
+                }
+            }
+            
+
+            var instances = DatabaseManager.World.GetCachedInstancesByLandblock(landblockId, variationId);
+            if (instances == null)
+            {
+                CommandHandlerHelper.WriteOutputInfo(session, $"Couldn't find landblock {landblockId:X4}");
+                return;
+            }
+
+            string sql_filename = string.Empty;
+            if (variationId.HasValue)
+            {
+                sql_filename = $"{landblockId:X4}_{variationId:N0}.sql";
+            }
+            else
+            {
+                sql_filename = $"{landblockId:X4}.sql";
+            }
+
+            try
+            {
+                if (LandblockInstanceWriter == null)
+                {
+                    LandblockInstanceWriter = new LandblockInstanceWriter();
+                    LandblockInstanceWriter.WeenieNames = DatabaseManager.World.GetAllWeenieNames();
+                }
+                MemoryStream mem = new MemoryStream();
```

Comment:
I recommend a using keyword when initializing variables for disposable objects to prevent memory leaks